src/index.ts

```diff
@@ -185,7 +185,8 @@ export class GraphQLServer {
       }
       return (response, ...args) => {
         if (
-          req.get('X-GraphQL-Deduplicate') &&
+          (req.get('X-GraphQL-Deduplicate') ||
+            req.query.deduplicate !== undefined) &&
```

Comment:
I'd check against `req.query.deduplicate` first given that this is now a preferred method. Perhaps, a comment about the legacy support of `X-GraphQL-Deduplicate` can be useful too.

I'm also wondering if there is a cheap way to be a bit smarter with handling the value and consider `?deduplicate`, `?deduplicate=true` and `?deduplicate=1` as `true`, while treat any other value as `false`. I'd be great if the implementation of this check could be reused for `graphql-crunch` too (https://github.com/graphcool/graphql-yoga/issues/303)

Reply:
I'd like such restriction for other values and it should be very simple to check.